blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -92,4 +92,5 @@ export const State = Record({
     steps: null,
     currentBranches: null,
     testResults: null,
+    capabilities: null,
```

Comment:
I see you forgot to remove that, however IMO you should re-insert the setCapabilities in the actions (just drop the fetch) and invoke the setCapabilities when you receive them from the store.